fluidsynth/src/drivers/fluid_alsa.c

```diff
@@ -347,19 +347,25 @@ static int fluid_alsa_handle_write_error (snd_pcm_t *pcm, int errval)
   case -EAGAIN:
     snd_pcm_wait(pcm, 1);
     break;
+// on some BSD variants ESTRPIPE is defined as EPIPE.
+// not sure why, maybe because this version of alsa doesnt support
+// suspending pcm stream. anyway, since EPIPE seems to be more 
+// likely than ESTRPIPE, so ifdef it out in case.
+#if ESTRPIPE != EPIPE
+  case -ESTRPIPE:
+    if (snd_pcm_resume(pcm) != 0) {
+      FLUID_LOG(FLUID_ERR, "Failed to resume the audio device");
+      return FLUID_FAILED;
+    }
+  /* fall through, since the stream got resumed, but still has to be prepared */
+#endif
```

Comment:
Yes, ESTRPIPE says the pcm stream is suspended and needs to be resumed(). I consider the condition `ESTRPIPE == EPIPE` an error in itself, dunno why such a define is possible at all. The only explanation is that these systems dont support suspended streams thus, ignore it at all, as EPIPE is more likely.